src/vm/siginfo.cpp

```diff
@@ -1403,7 +1403,7 @@ TypeHandle SigPointer::GetTypeHandleThrowing(
 
                 if (typeHnd.IsNull())
```

Comment:
There are other parsers of these signatures. e.g. `ZapSig::CompareSignatureToTypeHandle`. Do we need to make a matching change there as well?

Reply:
I'll take a look.